Formula/picotool.rb

```diff
@@ -0,0 +1,37 @@
+class Picotool < Formula
+  desc "Tool for interacting with a RP2040 device in BOOTSEL mode, or with a RP2040 binary"
+  homepage "https://github.com/raspberrypi/picotool"
+  url "https://github.com/raspberrypi/picotool/archive/refs/tags/1.1.1.tar.gz"
+  sha256 "2d824dbe48969ab9ae4c5311b15bca3449f5758c43602575c2dc3af13fcba195"
+  license "BSD-3-Clause"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libusb"
+
+  resource "pico-sdk" do
+    url "https://github.com/raspberrypi/pico-sdk/releases/download/1.5.0/sdk1.5.0-with-submodules.zip"
+    sha256 "59a09e619cab67b614d5f4e928a82f6bd055ce6083eb84f1b6caa269f1e3a559"
+  end
+
+  resource "pico-blink" do
+    url "https://rptl.io/pico-blink"
+    sha256 "4b2161340110e939b579073cfeac1c6684b35b00995933529dd61620abf26d6f"
+  end
+
+  def install
+    resource("pico-sdk").stage buildpath/"pico-sdk"
+
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args,
+                                              "-DPICO_SDK_PATH=#{buildpath}/pico-sdk"
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  test do
+    resource("pico-blink").stage do
+      system "#{bin}/picotool", "info", "blink.uf2"
+    end
+
+  end
```

Comment:
```suggestion
  test do
    resource("pico-blink").stage do
      system "#{bin}/picotool", "info", "blink.uf2"
    end
  end
```

Also, is it possible to validate its output with `assert_match` or similar?